Errror building nodejs-bindings on my windows-machine

## Bug description

While I am able to complete `cargo build` in a local wallet.rs-repo without any errors, `npm install @iota/wallet` fails on my windows machine with this error message:
`error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MT_StaticRelease'`
Full error message: https://ghostbin.com/paste/TgwaB

If I understand the error correctly, it means that my runtime lib is for some reason dynamic, while the project is supposed to be compiled statically?

## Versions

- Cmake  version 3.19.6
- Clang version 11.0.0
     Target: x86_64-pc-windows-msvc                                                                                          
     Thread model: posix                                                                                                     
     InstalledDir: D:\Program Files\LLVM\bin  

- Rust version 1.53
- node version 14.17
- openSSL version 1.1.1k


Which version of the library are you using?

- latest wallet-package version 0.5.0

## Hardware specification

Operating system:
- RAM: 16gb
- Cores: 8
- Device: Windows 10 Laptop

## Steps To reproduce the bug


This is most likely due to some form of local configuration on my system, as Ive been in contact with a few windows-users that have not faced any issues. 
1. Set cmd-directory to a npm-project
2. Enter `npm install @iota/wallet`

## Expected behaviour
Add package to projects without any errors

## Actual behaviour
Error while building nodejs bindings

## Errors

`error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MT_StaticRelease'`
Full error message: https://ghostbin.com/paste/TgwaB

--- a/bindings/nodejs/README.md
+++ b/bindings/nodejs/README.md
@@ -4,7 +4,7 @@ Node.js binding to the IOTA wallet library.
 
 ## Requirements (only for building the binary yourself)
 
-Ensure you have first installed the required dependencies for the library [here](https://github.com/iotaledger/wallet.rs/blob/dev/README.md) and on Windows also LLVM, our workflow uses `https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe`.
+Ensure you have first installed the required dependencies for the library [here](https://github.com/iotaledger/wallet.rs/blob/dev/README.md) and on Windows also LLVM, our workflow uses `https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe`. You might also need to set an environment variable `RUSTFLAGS` to `-C target-feature=+crt-static`.
 
 ## Installation
 

--- a/bindings/nodejs/README.md
+++ b/bindings/nodejs/README.md
@@ -4,7 +4,7 @@ Node.js binding to the IOTA wallet library.
 
 ## Requirements (only for building the binary yourself)
 
-Ensure you have first installed the required dependencies for the library [here](https://github.com/iotaledger/wallet.rs/blob/dev/README.md) and on Windows also LLVM, our workflow uses `https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe`. You might also need to set an environment variable `RUSTFLAGS` to `-C target-feature=+crt-static`.
+Ensure you have first installed the required dependencies for the library [here](https://github.com/iotaledger/wallet.rs/blob/dev/README.md) and on Windows also LLVM, our workflow uses `https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe`. On Windows, you may also need to set an environment variable `RUSTFLAGS` to `-C target-feature=+crt-static`.
 
 ## Installation
 

